extensions/spark/kyuubi-spark-connector-common/src/main/scala/org/apache/kyuubi/spark/connector/common/SparkUtils.scala

```diff
@@ -21,58 +21,15 @@ import org.apache.spark.SPARK_VERSION
 
 object SparkUtils {
```

Comment:
Should I delete this `SparkUtils` class?

Reply:
I'm neutral, either is fine.